checkov/ansible/checks/task/builtin/AptForce.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+from typing import Any
+
+from checkov.ansible.checks.base_ansible_task_value_check import BaseAnsibleTaskValueCheck
+from checkov.common.models.enums import CheckResult, CheckCategories
+
+
+class AptForce(BaseAnsibleTaskValueCheck):
+    def __init__(self) -> None:
+        name = "Ensure that packages with untrusted or missing signatures are not used"
```

Comment:
just thinking, if it makes sense to adjust the name here a bit to better differentiate between this and the other check, because using the `force` parameter also allows downgrading, which can result in broken state of incompatible package versions.

Reply:
Is the new wording getting too verbose?

Reply:
I know what you mean, but it makes it quite clear 🙂